docs/elevate/ELevating-CentOS6-to-CentOS7.md

```diff
@@ -0,0 +1,165 @@
+
+* When the migration process is over, reboot the system.
+* A new entry in GRUB called `System Upgrade (redhat-upgrade-tool)` will appear. The system will be automatically booted into it. See how the update process goes in the console. 
+  ::: tip
+  The system will reboot twice during the upgrade.
+  :::
+* After reboot, login to the system. Verify that the current OS is the one you need. 
+  ```sh
+  cat /etc/centos-release
+  ```
+  ###### Expected output:
+  ```sh
+  CentOS Linux release 7.2.1511 (Core) 
+  ```
+  
+### Clean up
+
+* Check packages left from the previous OS version, consider removing or updating them manually.
+  ```sh
+  rpm -qa | grep el6
+  ```
+* Check if any CentOS 6 repositories are left and disable them.
+* Consider removing the CentOS 6 upgrade repository package:
+  ```sh
+  yum remove elevate-release
+  ```
+  
+## Upgrade CenOS 7.2.1511 to CentOS 7.9.2009
+
+* As your *CenOS-Base.repo* file on CentOS 6 was earlier customized to contain CentOS Vault repositories, it wasn't updated during the migration process to contain CentOS 7 repositories. Instead, the new *CentOS-Base.repo.rpmnew* file was created. Run the following commands to replace the *.repo* file with the new one:
+  * ```sh
+    rpm -qV centos-release
+    ```  
+    ###### Expected output:
+    ```sh
+    S.5....T.  c /etc/yum.repos.d/CentOS-Base.repo
+    ```
+  * ```
+    mv /etc/yum.repos.d/CentOS-Base.repo.rpmnew /etc/yum.repos.d/CentOS-Base.repo
+    ```
+
+* Run the `yum` command to update your CentOS 7.2.1511 machine to the most recent version 7.9: 
+  ```sh
+  yum update
+  ```
+* Reboot after the update completes and verify the system's version: 
+  ```sh
+  cat /etc/centos-release
+  ```
+  ###### Expected output:
+  ```sh
+  CentOS Linux release 7.9.2009 (Core) 
+  ```
+  
+## Get Help and Contribute
+
+We welcome any feedback as it helps to improve the project.
+
+For any help and assistance, or if you want to share your thoughts or contribute to the ELevate project, reach out to us on the [AlmaLinux Community Chat](https://chat.almalinux.org/almalinux/channels/migration).
+   
+#### Additional documentation
+
+Here's the list of additional documentation that you might find useful:
+* [Red Hat Upgrade Tool](https://github.com/upgrades-migrations/redhat-upgrade-tool.git)
+* [A framework designed to run the Preupgrade Assistant modules](https://github.com/upgrades-migrations/preupgrade-assistant.git)
+* [Modules used by an upgrade from CentOS 6 to CentOS 7](https://github.com/upgrades-migrations/preupgrade-assistant-modules.git)
+* [CentOS Preupgrade Data](https://git.centos.org/sources/preupgrade-assistant-el6toel7-data/c6/)
+
+#### Trademarks
+Red Hat, Red Hat Enterprise Linux are trademarks or registered trademarks of Red Hat, Inc. or its subsidiaries in the United States and other countries.
+
+The CentOS Marks are trademarks of Red Hat, Inc. (“Red Hat”). 
```

Comment:
Not sure if I put the correct trademarks, @bennyvasquez @xsub mind taking a look?